k8s.tf

```diff
@@ -121,7 +122,7 @@ resource "kops_cluster" "k8s" {
       name = etcd_cluster.value
 
       dynamic "member" {
-        for_each = local.master_subnets_zones
+        for_each = toset([for k, v in var.control_plane.size : k if v.max > 0])
```

Comment:
Hmmm, should we use v.min > 0 here perhaps? If someone would configure their control planes as min 0 max 1? Or is max still correct here?